lavamoat/webpack/mv2/policy.json

```diff
@@ -1441,6 +1446,7 @@
         "@metamask/rpc-errors": true,
         "@metamask/safe-event-emitter": true,
         "@metamask/utils": true,
+        "deep-freeze-strict": true,
```

Comment:
This is a new dependency in `json-rpc-engine`, added in this PR: https://github.com/MetaMask/core/pull/6176